<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotstionTool</class>
 <widget class="QMainWindow" name="AnnotstionTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="LeftWidget" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="openFileButton">
           <property name="text">
            <string>Open File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveWorkButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save Work</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingsButton">
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="annotationGuideButton">
           <property name="text">
            <string>Annotation Guide</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="currentFileLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Current File:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="GLViewWidget" name="skeletonGraph">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="reverseFastButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string> Fast Reverse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reverseButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Reverse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reverseSlowButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Slow Reverse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playPauseButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forwardSlowButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Slow Forward</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forwardButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Forward</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forwardFastButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Fast Forward</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Input Frame:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="currentFrameLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>73</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setStartPointButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Set to start point</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="frameSlider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="currentFrameLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Current Frame: -/-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="RightWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.opengl</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
